lib/elasticsearch_relation.rb

```diff
@@ -20,7 +19,7 @@ def all
   end
 
   def count
-    Annotation.search(@params.merge({ size: 0 })).total
+    Annotation.search(@params).total
```

Comment:
I tested that the number set in `size` has no effect on the `total` result. E.g. I set `size` to 10 and it still returned a correct total of 14.